File: 861-FlippingAnImage/861-FlippingAnImage.py
# Last updated: 7/2/2025, 11:52:02 PM
class Solution:
    def flipAndInvertImage(self, image: List[List[int]]) -> List[List[int]]:
        res = []
        for i in range(len(image)):
            l = []
            for k in image[i]:
                l.append(k)
                
            l.reverse()
            for j in range(0,len(l)):
                if l[j] == 0:
                    l[j]=1
                else:
                    l[j]=0
            res.append(l)
        return res
        
                       

File: 861-FlippingAnImage/861-FlippingAnImage.cpp
// Last updated: 7/2/2025, 11:51:55 PM
class Solution {
public:
    vector<vector<int>> flipAndInvertImage(vector<vector<int>>& image) {
        for(auto& img : image)
        {
            reverse(img.begin(), img.end());

            for(int& pixel : img)
            {
                pixel = 1 - pixel;
            }
        }
        return image;
    }
};